src/SmartTransactionsController.test.ts

```diff
@@ -451,6 +524,36 @@ describe('SmartTransactionsController', () => {
         tradeTxFees: getFeesApiResponse.txs[1],
       });
     });
+
+    it('should add fee data to feesByChainId state using the networkClientId passed in to identify the appropriate chain', async () => {
+      const tradeTx = createUnsignedTransaction(goerliChainIdDec);
+      const approvalTx = createUnsignedTransaction(goerliChainIdDec);
+      const getFeesApiResponse = createGetFeesApiResponse();
+      nock(API_BASE_URL)
+        .post(`/networks/${goerliChainIdDec}/getFees`)
+        .reply(200, getFeesApiResponse);
+
+      expect(
+        smartTransactionsController.state.smartTransactionsState.feesByChainId,
+      ).toStrictEqual(defaultState.smartTransactionsState.feesByChainId);
```

Comment:
is this needed? I guess the reason why you're doing this is because you're not confident that outside polling hasn't changed this for us?

Reply:
yeah def not strictly needed, I just tend to add these extra assertions to make sure that the later assertions are still testing what they think they are.